modules/rostests/winetests/bcrypt/CMakeLists.txt

```diff
@@ -2,5 +2,8 @@
 remove_definitions(-D_WIN32_WINNT=0x502)
 add_executable(bcrypt_winetest bcrypt.c testlist.c)
 set_module_type(bcrypt_winetest win32cui)
-add_importlibs(bcrypt_winetest bcrypt advapi32 user32 msvcrt kernel32)
+# bcrypt is available on WVista (and ReactOS), not on WS03.
+# advapi32/RegGetValueW() is available on WXP-Pro-x64 and WS03sp1, not on WXP.
+add_delay_importlibs(bcrypt_winetest bcrypt advapi32)
```

Comment:
There's no reason to make advapi32 a delay import, even with the current workaround.

Reply:
Agreed. I simply tried to minimize the test fork. I shall make this purely dynamic, then.